modules/core-updates/9.0.md

```diff
@@ -85,6 +85,40 @@ Some dependencies are still present but were upgraded which comes with their own
+
+The login page to the back-office has been migrated to Symfony, along ith it the system of authorization in the BackOffice is now also based on Symfony, it implies several things:
+- we no longer depend on the legacy `Context::$cookie`, the session is kept on the server side and very few data are kept on the browser side, for retro-compatibility we still populate the legacy cookie so that you can **read** it but modifying/writing it will likely no longer have the effect you would expect (if any)
+- the `PrestaShopBundle\Security\Admin\EmployeeProvider` and `PrestaShopBundle\Service\DataProvider\UserProvider` now return a `PrestaShopBundle\Entity\Employee` instance, their responsibility as to return a `Symfony\Component\Security\Core\User\UserInterface` anyway (and they still do) but in case you depended on the child class know that `PrestaShopBundle\Security\Admin\Employee` no longer exist
+- if you need to get the logged-in user you can use the Symfony `Symfony\Bundle\SecurityBundle\Security` service, but we recommend you'd rather use the `EmployeeContext` internal PrestaShop service instead (see explanation about [new Contexts]({{< relref "9/modules/core-updates/9.0.md#symfony-context-refacto" >}}))
+
+**Removed hooks:**
+- `actionAdminLoginControllerBefore`
+- `actionAdminLoginControllerLoginBefore`
+- `actionAdminLoginControllerLoginAfter`
+- `actionAdminLoginControllerForgotBefore`
+- `actionAdminLoginControllerForgotAfter`
+- `actionAdminLoginControllerResetBefore`
+- `actionAdminLoginControllerResetAfter`
+
+**Hooks kept for backward compatibility:**
+- `actionAdminLoginControllerSetMedia` so you can add some custom assets on the login page
+- `displayAdminLogin` so you can add custom HTML on the login page
+
+**New hooks**
+- `actionBackOfficeLoginForm` to modify the login form (the form builder is passed via the `form_builder` hook parameter)
+- `actionEmployeeRequestPasswordResetForm` to modify the request password form (the form builder is passed via the `form_builder` hook parameter)
+
+The logic of the login, however, is not handled by PrestaShop internal code so there is no handler that interprets the submitted data (and that's hy there is so few new hooks), instead we rely on <a href="https://symfony.com/doc/current/security.html#form-login" target="_blank">Symfony form login authentication system</a>, so it's Symfony internal system that interprets the data submitted, validates it and authenticate the employee to the back-office.
+
+If you need to interface your code with the authentication process you should now rely on the <a href="https://symfony.com/doc/current/security.html#authentication-events" target="_blank">Authentication events</a> described in their documentation, you can also inspire yourself from our <a href="https://github.com/PrestaShop/PrestaShop/blob/develop/src/PrestaShopBundle/EventListener/Admin/EmployeeSessionSubscriber.php" target="_blank">own internal subscriber</a>.
```

Comment:
```suggestion
If you need to interfere with the authentication process, you should now rely on the <a href="https://symfony.com/doc/current/security.html#authentication-events" target="_blank">Authentication events</a> described in their documentation, you can also inspire yourself from our <a href="https://github.com/PrestaShop/PrestaShop/blob/develop/src/PrestaShopBundle/EventListener/Admin/EmployeeSessionSubscriber.php" target="_blank">own internal subscriber</a>.
```

Reply:
I prefered the word `integrate`, interfere makes it feel like you wanna prevent the initial behaviour more than add something to it